libs/community/langchain_community/utilities/steam.py

```diff
@@ -156,9 +157,25 @@ def recommended_games(self, steam_id: str) -> str:
         return str(top_5_popular_not_owned)
 
     def run(self, mode: str, game: str) -> str:
+        """Run Steam API operations.
+
+        Args:
+            mode (str): Operation mode, can be either 'get_games_details' or 'get_recommended_games'
+            game (str): Game name or Steam ID, depending on the operation mode
+
+        Returns:
+            str: Result information of the operation
+
+        Raises:
+            ValueError: When an invalid operation mode is provided
+        """
         if mode == "get_games_details":
             return self.details_of_games(game)
         elif mode == "get_recommended_games":
-            return self.recommended_games(game)
+            try:
+                from decouple import config
+            except ImportError:
+                raise ImportError("decouple library is not installed. ")
+            return self.recommended_games(config("STEAM_ID"))
```

Comment:
If we're already fixing this code, could we remove dependency on `decouple` and instead access STEAM_ID from the env?